examples/Physics/physics.js

```diff
@@ -0,0 +1,280 @@
+var PhysicsApplication = ICameraApplication.$extend(
+{
+    __init__ : function()
+    {
+        this.$super("Physics");
+
+        // Enables animations if both the previous and this camera entity is unparented
+        this.animateBeforeActivation(true);
+        
+        this.showTooltip = TundraSDK.framework.frame.wallClockTime() + 3;
+        this.tooltipVisible = false;
+        
+        this.demoApp = {
+            origin: new THREE.Vector3(0, -10, -30),
+            world : {},
+            boxes : [],
+            projectiles : [],
+            
+            createScene : function()
+            {
+                this.world["Ground"] = this._createMesh("Ground", "webtundra://plane.json");
+                this.world["Ground"].name = "Ground";
+                t = this.world["Ground"].placeable.transform;
+                t.setPosition(this.origin.x, this.origin.y, this.origin.z);
+                t.setScale(100, 2, 100);
+                this.world["Ground"].placeable.transform = t;
+                this.world["Ground"].rigidbody.mass = 0.0;
+                
+                this.world["Temp"] = this._createMesh("Temp", "webtundra://Box.json");
+                this.world["Temp"].name = "Temp";
+                t = this.world["Temp"].placeable.transform;
+                t.setPosition(1000, 1000, 1000);
+                this.world["Temp"].placeable.transform = t;
+                
+                this.world["Temp2"] = this._createMesh("Temp", "webtundra://Sphere.json");
+                this.world["Temp2"].name = "Temp2";
+                t = this.world["Temp2"].placeable.transform;
+                t.setPosition(1000, 1000, 1000);
+                this.world["Temp2"].placeable.transform = t;
+                
+                this.world["Temp"].removeComponent(this.world["Temp"].rigidbody.id);
+                this.world["Temp2"].removeComponent(this.world["Temp2"].rigidbody.id);
+                
+                this.world["DirLight"] = new THREE.DirectionalLight();
+                this.world["DirLight"].intensity = 0.5;
+                TundraSDK.framework.client.renderer.scene.add(this.world["DirLight"]);
+            },
+            
+            createBoxes : function()
+            {
+                var tilling = 4;
+                var offset = -2 * tilling;
```

Comment:
useless nitpick :p .. but anyhow: `tilling` is probably not what you mean here but perhaps `tiling`?